Stratis.Bitcoin.IntegrationTests/RPCTEsts.cs

```diff
@@ -1,7 +1,12 @@
-﻿using NBitcoin;
+﻿using System;
```

Comment:
This class is called RPCTEsts.cs with a capital E at TEsts. Can you please rename it?